### PR TITLE
NEWS.md: update for 0.7.2

### PR DESCRIPTION
**Pull request by johanmalm**
For Friday

### CODE REVIEW

**Comment by tokyo4j** [comment]
I suggest `- Prevent Chromium from crashing when started after a virtual keyboard is destroyed. #1789`.

**Comment by johanmalm** [comment]
> I suggest `- Prevent Chromium from crashing when started after a virtual keyboard is destroyed. #1789`.

Thanks. Done.

File: NEWS.md [hunk 1]
```diff
@@ -1326,7 +1345,8 @@ Compile with wlroots 0.12.0 and wayland-server >=1.16
   ShowMenu
 
 [Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
-[unreleased]: https://github.com/labwc/labwc/compare/0.7.1...HEAD
+[unreleased]: https://github.com/labwc/labwc/compare/0.7.2...HEAD
+[0.7.2]: https://github.com/labwc/labwc/compare/0.7.0...0.7.2
```
Comment (Consolatis) [inline NEWS.md:1349]: this should likely compare to 0.7.1 rather than 0.7.0

Comment (johanmalm) [reply NEWS.md:1349]: Thanks for catching. Will change tomorrow.

**Pull request merged**